Percolation.java

```diff
@@ -1,14 +1,135 @@
 import edu.princeton.cs.algs4.StdRandom;
-import edu.princeton.cs.algs4.StdStats;
+import  edu.princeton.cs.algs4.StdStats;
 import edu.princeton.cs.algs4.WeightedQuickUnionUF;
 
 public class Percolation {
-   public Percolation(int n)                // create n-by-n grid, with all sites blocked
-   public    void open(int row, int col)    // open site (row, col) if it is not open already
-   public boolean isOpen(int row, int col)  // is site (row, col) open?
-   public boolean isFull(int row, int col)  // is site (row, col) full?
-   public     int numberOfOpenSites()       // number of open sites
-   public boolean percolates()              // does the system percolate?
-
-   public static void main(String[] args)   // test client (optional)
-}
\ No newline at end of file
+
+  // UF used for percolates test
+  private WeightedQuickUnionUF PercolateTest;   
+
+  // UF used for connection site test
+  // open site in the top row
+  private WeightedQuickUnionUF TestFull;
+
+  // siteStatus stores every site's open or block
+  // true means open,false means block
+  private boolean[] siteStatus;
+
+  // percolation experiment's grid width and height
+  private int SIZE;
```

Comment:
This variable is not static final (constant), then shouldn't be written in capital letters... Same for the two bellow...